view/templates/admin_site.tpl

```diff
@@ -126,6 +126,7 @@
 	{{include file="field_input.tpl" field=$basepath}}
 	{{include file="field_checkbox.tpl" field=$suppress_tags}}
 	{{include file="field_checkbox.tpl" field=$nodeinfo}}
+	{{include file="field_select.tpl" field=$checknewversionurl}}
```

Comment:
Standards: Please use `underscore_case` for the `$checknewversionurl` variable name.